### DISCUSSION TITLE
Cloud Saves

### DISCUSSION THREAD

**Discussion by Schnubbirunkel** [diasurgical/DevilutionX#5425] [Ideas]
I don't know, how hard it is, but would it be possible to include cloud saving over platforms like Dropbox, Google Drive etc...?
I love playing on the go with my Steam Deck or Laptop, but when I'm at home, PC is the way to go, so Cloud Saves would be so awesome for that.
Maybe something like ScummVM does it. That really works great on cross-platform.

**Comment by AJenbo** [comment]
It has been suggested by a few people at this point. It's not something that would be impossible, but:
- None of the regular developers are particularly interested in this feature, or maintaining it
- It's not impossible to do, but it's probably a lot of work (ScummVM's is +7000 lines of code)
- We can't reuse the one from ScummVM because of conflicting licenses

So what you need is to find someone interested in developing this feature.
